topostats/plottingfuncs.py

```diff
@@ -38,7 +38,7 @@ def __init__(
         image_type: str = "non-binary",
         image_set: str = "core",
         core_set: bool = False,
-        interpolation: str = "nearest",
+        interpolation: str = None,
```

Comment:
If you truly want to eliminate any user control of this then the option should be hard-coded in the call(s) to `plt.imshow()`. As it stands if someone were to use the class interactively they could still change the value.

Reply:
Sounds good - I've moved it to be hardcoded in the calls! I think as we've already got the gaussian function in the flattening section which applies the filter to the image, it essentially will do the same thing so I don't think there is a need for it here. What do you think?

Reply:
The flattening is part of the image processing pathway (and there is also Gaussian methods applied as part of the DNA Tracing processing).

The plotting is very much separate from that in so much as it could be used to plot a `np.ndarray` at any point. I prefer to design software to not have hard-coded values but rather to be open and as flexible as possible. Currently this works through `**kwargs`.

One thing I'd ask if the open and flexible design of making this a configurable option is to be ditched is how likely people are to...

* go with anything but the default settings that are loaded.
* if using a custom configuration file change the field pertaining to `interpolation` method?

I realise these are somewhat rhetorical, but is there any point taking away configuration options because they shouldn't be used if there is a very low likelihood that they will be changed?

Making it configurable but clear what the developers recommend and why that recommendation is made would be my preference.

Reply:
Makes sense. I'll revert the changes (and learn how to do so in the process!)